Automattic/woocommerce-payments
# Upgrade docker image to PHP 8.1

Fixes #9092 

#### Changes proposed in this Pull Request

* Upgrades the local development environment to use PHP 8.1.
* Updates PHP coverage tests on GitHub actions to run on both PHP 7.4 & 8.1
* Updates WordPress stubs to 5.9.6
* Fixes PSALM errors

#### Testing instructions

* On you local machine, install & switch to PHP 8.1. If you're using homebrew you can install PHP 8.1 using `brew install php@8.1` or using https://github.com/shivammathur/homebrew-php.
* Checkout this branch & run `npm install` to install dependencies.
* Run `npm run up` to spin up docker containers with the new image.
* Run `npm run cli` to connect to docker shell and run `php --version` to confirm the PHP version as 8.1
* Confirm the following scripts works as expected.
  * `npm run build`
  * `npm run test`
  * `npm run test:php-coverage`
  * `npm run lint:php`
  * `npm run psalm`
* On this PR, confirm all the workflows works as expected.

-------------------


**Post merge**
